src/jquery.multiselect.js

```diff
@@ -917,12 +1119,12 @@
 
       var overflowSetting = 'hidden';
       var itemCount = 0;
-      var ulHeight = 0;  // Adjustment for hover height included here.
+      var ulHeight = 4;  // Adjustment for hover height included here.
 
       // The following adds up item heights.  If the height sum exceeds the option height or if the number
       //   of item heights summed equal or exceed the native select size attribute, the loop is aborted.
       // If the loop is aborted, this means that the menu must be scrolled to see all the items.
-      $checkboxes.find('li,a').each( function() {
+      self.$checkboxes.find('li:not(.ui-multiselect-optgroup),a').filter(':visible').each( function() {
```

Comment:
Why exclude the option groups here?

Reply:
Got it, since find also goes into descendants. Makes sense.

Reply:
@mlh758 If you have any thoughts on any better ways to account for the heights of the option group headings, I am all ears.  As I noted, I do not think the current method is perfect.

Reply:
I don't at this time, I just wanted to know why the change was made since it was so recently added.

Reply:
The main potential problem here is if someone is using collapsable groups, then you also have the height of the collapse buttons to consider.  If someone changes the CSS to make the buttons taller, then potentially the button heights should be examined instead of the anchor heights.   Right now, I am not seeing much of a difference, but it has occurred to me that if the element being examined is an anchor, maybe I should look for a sibling button and use the Math.max height of the two (button and anchor).